data/core/macros/abilities.cfg

```diff
@@ -387,3 +387,18 @@ Enemy units cannot see this unit while it is in deep water, except if they have
         special_note={INTERNAL:SPECIAL_NOTES_FEEDING}
     [/dummy]
 #enddef
+
+#define ABILITY_EXPLODE TYPE VALUE
+    # Canned definition of the Explode ability to be included in an
+    # [abilities] clause.
+    # This works because of data/lua/explode_ability.lua
+    [explode]
+        id=explode
+        name= _ "explode"
+        female_name= _ "female^explode"
+        description=_ "This unit explodes upon death, causing damage to any adjacent units."
+        value={VALUE}
+        apply_to={TYPE}
```

Comment:
since apply_to already has a meaning in weapon specials (self/opponent) i this k it might be better if this key had a different name.

Reply:
That's a good point… why not just call it `damage_type`?

Reply:
OK, so the engine will change.  Master has developed since this was last touched and I hope to make the animation better anyway, so I'll close this PR.  I'll try again when the requirements are more firm.